src/engine/tools/engine-suite/src/engine_test/parser.py

```diff
@@ -0,0 +1,42 @@
+from enum import Enum
+import sys
+
+class Parser:
```

Comment:
improve - desing: You shouldn't call Parser inside Parser, you're missing the self pointer